install.sh

```diff
@@ -223,10 +251,21 @@ main () {
         echo
         info "No Dune version was specified, so the installer will check the latest binary release of Dune..."
         echo
-        version=$(latest_binary_dune_version)
+        version=$(latest_binary_dune_version "${debug_version_repo:-"$dune_bin_git_url"}" "$allow_unstable")
         echo
         info "The latest binary release of Dune was found to be $version."
         echo
+    elif [ "$allow_unstable" = "1" ]; then
+        echo
+        error "--allow-unstable has no impact when version number is explicitly specified"
```

Comment:
I'm a bit confused by this -- does it mean if I want to install an older unstable version, I just need to specify it in the version and don't need the --allow-unstable flag?

Reply:
If you don't pass the install script a version number then it will install the latest version of dune. This flag allows the installer to consider unstable versions of dune when choosing the latest version. If you want to install an specific unstable version of dune you can always do so by passing that version to the script explicitly though. Perhaps we should just prevent the script from automatically choosing an unstable version altogether rather than providing that behaviour behind a flag. What do you think?